src/main/java/org/mockito/Mock.java

```diff
@@ -111,4 +112,12 @@
      * @since 2.23.3
      */
     boolean lenient() default false;
```

Comment:
We should deprecate this method as well.

Reply:
Added deprecation marker and suggestion to use `strictness()`.